static/prelude_canonical.rs

```diff
@@ -0,0 +1,273 @@
+use cbor_event::{self, de::Deserializer, se::Serializer};
```

Comment:
I tried to programmatically generate the differences using the codegen crate, but unfortunately non-default implementations of trait functions are not supported in the latest version so I ended up just having two different prelude files instead. It would have been nice to be able to do it programmatically though since then you could only output the parts of this prelude that are needed.